src/Elsa.CustomActivities/Activities/Shared/QuestionBookmarkProvider.cs

```diff
@@ -43,7 +43,8 @@ private List<string> RegisteredActivityTypes()
             {
                 ActivityTypeConstants.QuestionScreen,
                 ActivityTypeConstants.CheckYourAnswersScreen,
-                ActivityTypeConstants.ConfirmationScreen
+                ActivityTypeConstants.ConfirmationScreen,
+                ActivityTypeConstants.PotScoreCalculation,
```

Comment:
I think this can actually go now, as it should only be relevant for blocking activities

Reply:
Just tested locally and it's fine without it.  Pushed up the changes.